test/test_browser.py

```diff
@@ -2197,7 +2197,9 @@ def test_cube_explosion(self):
 
   @requires_graphics_hardware
   def test_glgettexenv(self):
-    self.btest('glgettexenv.c', args=['-sLEGACY_GL_EMULATION', '-lGL', '-lSDL'], expected='1')
+    # Test with 4GB maximum memory to test that GL emulation does not error
+    # during link.
+    self.btest('glgettexenv.c', args=['-sLEGACY_GL_EMULATION', '-lGL', '-lSDL', '-sMAXIMUM_MEMORY=4GB', '-sALLOW_MEMORY_GROWTH'], expected='1')
```

Comment:
You would also get the same error by referencing these symbols in a wasm64 build.

Reply:
Would that be better? It seems less intrusive to increase the max memory to me. Unless you mean we should also have wasm64 coverage?

Reply:
No, this approach seems fine.